cmd/pggen/test/tables_test.go

```diff
@@ -1116,6 +1116,35 @@ func TestNotFoundList(t *testing.T) {
 	}
 }
 
+func TestListSucceedOnPartial(t *testing.T) {
+	txClient, err := pgClient.BeginTx(ctx, nil)
+	chkErr(t, err)
+	defer func() {
+		_ = txClient.Rollback()
+	}()
+
+	entity := models.SmallEntity{
+		Anint: 1892,
+	}
+	smallEntityID, err := txClient.InsertSmallEntity(ctx, &entity)
+	chkErr(t, err)
+
+	params := [][]int64{
+		{23423, smallEntityID}, // partial match
+		{23423},                // just completely missing
+	}
+
+	for _, p := range params {
+		// a partial match will be tagged with NotFoundError
+		entities, err := pgClient.ListSmallEntity(ctx, p)
```

Comment:
It doesn't look like we are actually passing in the new functional option, so I don't think we're really testing it.

Reply:
🤦‍♂️